lib/calculable.rb

```diff
@@ -0,0 +1,7 @@
+module Calculable
```

Comment:
Deliberately returning an un-rounded float. In multiple cases the methods calling on this need to do further work on the output (for example, multiplying by 100 to get a percent value) before rounding.